client/webpack.client.base.config.js

```diff
@@ -0,0 +1,67 @@
+// Common client-side webpack configuration used by webpack.hot.config and webpack.rails.config.
+
+const webpack = require('webpack');
+const path = require('path');
+
+const devBuild = process.env.NODE_ENV !== 'production';
+const nodeEnv = devBuild ? 'development' : 'production';
+
+module.exports = {
+
+  // the project dir
+  context: __dirname,
+  entry: {
+
+    // See use of 'vendor' in the CommonsChunkPlugin inclusion below.
+    vendor: [
+      'babel-core/polyfill',
+      'jquery',
+      'jquery-ujs',
+      'react',
+      'react-dom',
```

Comment:
If you don't put `react` and `react-dom` as an entry point, then they might not get exposed on the line below: https://github.com/shakacode/react_on_rails-generator-results/pull/123/files#diff-7b11f5a99db3faafe8e1a74d1f1d5df0R62